packages/gatsby/src/redux/__tests__/jobs.ts

```diff
@@ -29,16 +30,15 @@ describe(`Job actions/reducer`, () => {
   })
 
   it(`throws an error if an ID isn't provided`, done => {
-    function runReducer() {
```

Comment:
Can you please undo the function->arrow changes? (Also below) Or explain why you're doing them?

Reply:
Since the `this` keyword is not used, I thought about removing them.
You are correct that there is no benefit to this change, I'm removing them :+1: .